src/utilities/http/fillInErrorWithDefaults.ts

```diff
@@ -8,8 +8,10 @@ export const fillInErrorWithDefaults = (error: Partial<HttpErrorResponseModel>,
     status: error.status || 0,
     message: error.message || 'Error requesting data',
     url: error.url || request.url,
-    raw: error.raw,
     // Remove anything with undefined or empty strings.
     errors: errors.filter(isDefined),
+    // TODO: figure out what to pass for raw because a circular-structure was happening on errors
+    //  because I held onto the same error object that next.js gave me.
+    // raw: error.raw,
```

Comment:
https://github.com/zeit/next.js/blob/master/errors/circular-structure.md

circular-structure was happing on errors because I held onto the same error object that next.js gave me.